src/SonarScanner.MSBuild.PreProcessor/Roslyn/RoslynAnalyzerProvider.cs

```diff
@@ -140,30 +132,30 @@ private IEnumerable<AnalyzerPlugin> FetchAnalyzerPlugins(string language, IEnume
 
         if (plugins.Count == 0)
         {
-            this.logger.LogInfo(Resources.RAP_NoAnalyzerPluginsSpecified, language);
-            return Enumerable.Empty<AnalyzerPlugin>();
+            logger.LogInfo(Resources.RAP_NoAnalyzerPluginsSpecified, language);
+            return [];
         }
         else
         {
-            this.logger.LogInfo(Resources.RAP_ProvisioningAnalyzerAssemblies, language);
-            return this.analyzerInstaller.InstallAssemblies(plugins);
+            logger.LogInfo(Resources.RAP_ProvisioningAnalyzerAssemblies, language);
+            return analyzerInstaller.InstallAssemblies(plugins);
         }
     }
 
     private static ICollection<string> ActiveRulesPartialRepoKeys(IEnumerable<SonarRule> rules)
     {
         var partialRepoKeys = new HashSet<string>
         {
-            // Always add SonarC# and SonarVB to have at least tokens...
-            string.Format(SONARANALYZER_PARTIAL_REPO_KEY, "cs"),
-            string.Format(SONARANALYZER_PARTIAL_REPO_KEY, "vbnet")
+            // Always add C# and VB.NET to have at least tokens...
+            string.Format(SonarAnalyzerRepoKey, "cs"),
```

Comment:
This is quite obfuscated way to do this
```suggestion
            SonarAnalyzerRepoPrefix + "cs",
```